dev/mongo-rs-backups/rs-bootstrap.js

```diff
@@ -0,0 +1,34 @@
+var conn, attempt;
+while (conn === undefined) {
+    try {
+        conn = new Mongo("localhost:27017");
+    } catch (Error) {
+        attempt++;
+    }
+
+    if (attempt >= 10) {
+        print("Max connection attempts exceeded.");
+        break;
+    }
+    sleep(100);
```

Comment:
10 times 100 would make only one second. Is that OK to try to connect 10 times within one second?